app/models/favorite_handler.rb

```diff
@@ -0,0 +1,13 @@
+class FavoriteHandler < ActiveRecord::Base
```

Comment:
I'm curious why you went with `FavoriteHandler` here instead of simply `Favorite`